automated-comments/go/hamming/invert_if_for_happy_path_on_left.md

```diff
@@ -0,0 +1,4 @@
+You could make the function more readable by inverting the condition of the `if` statement handling the error case. 
+The rest of the function can then be outdented because an `else` branch is not needed. A Go proverb 
+says "The happy path is left-aligned", meaning the normal flow of control follows the left-hand margin, and only 
+exceptional or error cases are indented.
```

Comment:
I really like the Go proverb part in the second sentence. I think we can tighten up the first with something like:

> To simplify, invert the `if` condition handling the error case, then get rid of the `else` branch by returning within the `if`.

Reply:
Suggesting to add this link on "[The happy path is left-aligned](https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88)".

Reply:
Nice! :)